adapters/feedad/feedad.go

```diff
@@ -0,0 +1,105 @@
+package feedad
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/prebid/openrtb/v20/openrtb2"
+	"github.com/prebid/prebid-server/v3/adapters"
+	"github.com/prebid/prebid-server/v3/config"
+	"github.com/prebid/prebid-server/v3/errortypes"
+	"github.com/prebid/prebid-server/v3/openrtb_ext"
+	"github.com/prebid/prebid-server/v3/util/jsonutil"
+)
+
+const feedAdAdapterVersion = "1.0.0"
+
+func getHeaders(request *openrtb2.BidRequest) http.Header {
+	headers := http.Header{}
+	headers.Add("Accept", "application/json")
+	headers.Add("Content-Type", "application/json;charset=utf-8")
+	headers.Add("X-FA-PBS-Adapter-Version", feedAdAdapterVersion)
+	headers.Add("X-Openrtb-Version", "2.5")
```

Comment:
Are you able to support OpenRTB 2.6 instead?

Reply:
I’m afraid, we don’t have support for oRTB 2.6, yet, but it’s on our roadmap for next year.
Can we start with oRTB 2.5 for now?